prac_05/state_names.py

```diff
@@ -0,0 +1,21 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+print(CODE_TO_NAME)
+
+for state_code in CODE_TO_NAME:
+    print(f"{state_code:3} is {CODE_TO_NAME[state_code]}")
+
+state_code = input("Enter short state: ").upper()
+while state_code != "":
+    if state_code in CODE_TO_NAME:
+        print(state_code, "is", CODE_TO_NAME[state_code])
+    else:
+        print("Invalid short state")
+    state_code = input("Enter short state: ").upper
```

Comment:
You could say this the prompt for the user again for another state code input.

Reply:
bro the task was to reformate this code. this is not my code its example code. but thanks for letting me know 😘😘😘